packages/graph/hash_graph/lib/graph/src/store/postgres/query/table.rs

```diff
@@ -112,14 +109,18 @@ pub enum ColumnAccess<'q> {
     /// Accesses a field of a JSON blob: `"column"->>'field'`
     Json {
         column: &'static str,
-        field: Cow<'q, str>,
+        field: &'q str,
     },
+    /// Accesses a field of a JSON blob: `"column"->>'field'`
```

Comment:
nit:
This docstring is a copy paste of the one above :)

Reply:
Fixed in https://github.com/hashintel/hash/pull/1230/commits/f23961655f68eb9720395596420636e063507964